src/components/feedback/Snackbar/index.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import styles from './styles.less';
+import classNames from 'classnames';
+
+export enum verticalPositions {
+    top = 'top',
+    bottom = 'bottom',
+}
+export enum horizontalPositions {
+    left = 'left',
+    right = 'right',
+    center = 'center',
+}
+
+type SnackBarProps = {
+    verticalPosition?: verticalPositions;
+    horizontalPosition?: horizontalPositions;
+    open?: boolean;
+    autoHideDuration?: number;
+    onClose?: () => void;
+    message?: string;
+    actions?: Array<JSX.Element>;
```

Comment:
According to the specification the snackbar allows 0-1 action(s). Should we, instead of accepting an array of elements, accept a single string?
E.g.
`
cancellable: true,
cancelLabel: "Undo",
onCancel: () => undoDelete();
`